publisher/pubsub_connection.go

```diff
@@ -86,22 +84,17 @@ func (c *PubSubConnection) Close() error {
 }
 
 type retryPausedResult struct {
-	inner       *pubsub.PublishResult
-	topic       *pubsub.Topic
-	data        []byte
-	orderingKey string
+	inner *pubsub.PublishResult
+	topic *pubsub.Topic
+	data  []byte
 }
 
 func (r *retryPausedResult) Get(ctx context.Context) (string, error) {
 	serverID, err := r.inner.Get(ctx)
 
 	if err != nil && errors.Is(err, pubsub.ErrPublishingPaused{}) {
-		// if we failed to publish because publishing is paused, resume publishing and retry
-		r.topic.ResumePublish(r.orderingKey)
```

Comment:
I guess this is _only_ specific to ordering keys, eh? i.e., there's no pause/resume otherwise?